stable/openebs/templates/daemonset-ndm.yaml

```diff
@@ -8,7 +8,10 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     component: ndm
+    openebs.io/component-name: ndm
 spec:
+  updateStrategy:
+    type: "{{ .Values.ndm.updateStrategy.type }}"
```

Comment:
this might break current deployments.
did you make a test to validate?

Reply:
Since the older releases had type:OnDelete - changing to default value of type:RollingUpdate will work. However, if the type is set to RollingUpdate with the current version and in future, the values.yaml is modified to set any other type - like Recreate etc., then this will cause issues with upgrade. 

@prateekpandey14  - if you don't want the users to modify the upgrade type, can you set the specific value?

Reply:
Yes make sense, updated the policy to specifically to use  `RollingUpdate` and not being configurable.
Thanks @cpanato for pointing out.